language/stdlib/modules/libra_coin.mvir

```diff
@@ -39,7 +39,7 @@ module LibraCoin {
         // this is to prevent the market cap's total value from hitting u64_max due to excessive
         // minting. This will not be a problem in the production Libra system because coins will
         // be backed with real-world assets, and thus minting will be correspondingly rarer.
-        assert(copy(value) <= 1000000000 * 1000, 11); // * 1000 because the unit is microlibra
+        assert(copy(value) <= 1000000 * 1000000, 11); // * 1000000 because the unit is microlibra
```

Comment:
```
assert(copy(value) <= 1_000_000_000 * 1_000_000, 11);
```

To match the comment string talking about 1B limit

Reply:
Yeah, either 1 million or 1 billion should be fine, as long as the comment and code are consistent... You might not be able to do `1_000` since this is Move, not Rust...

Reply:
True, I forgot

Reply:
Will fix locally.